src/univariate/discrete/binomial.jl

```diff
@@ -73,6 +73,22 @@ function mode(d::Binomial{T}) where T<:Real
 end
 modes(d::Binomial) = Int[mode(d)]
 
+function median(rv::Binomial)
+    bound = min(rv.p, 1-rv.p)
+    rv_mean = mean(rv)
+    floor_mean = floor(Int, rv_mean)
+    ceil_mean = ceil(Int, rv_mean)
+    if abs(floor_mean - rv_mean) <= bound
+        floor_mean
+    elseif abs(ceil_mean - rv_mean) <= bound
```

Comment:
This seems to be equivalent to
```suggestion
    elseif rv_mean - floor_mean >= 1 - bound
```
which could be written simpler and more efficient by computing `rv_mean - floor_mean` only once and calculating `minmax(d.p, 1 - d.p)` instead of only `min`.

Reply:
I made a change that seems to work with `min` only and without needing to recompute `rv_mean - floor_mean`